src/modules/ipfix/aggregator/HashtableBuckets.h

```diff
@@ -32,8 +32,8 @@ class BucketListElement;
 class HashtableBucket
 {
 public:
-	uint32_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
-	uint32_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
+	time_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
```

Comment:
Done all renames by grep | xargs sed. It's a big-ish diff and touches many file, let me know if it's fine for you.